### PR TITLE
DOC-5592: Updated Signing Packages Using CLI steps

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by anien — APPROVED** [review]
Thanks!

File: content/en/platform/corda/5.0/developing-applications/packaging/code-signing/signing-cli.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ mycpb.cpb \
 --key "signing key 1"
 ```
 
-2. Supply the group policy into CPI version 2 by performing one of the following actions:
+2. Build the CPI. Supply the group policy into CPI version 2 by performing one of the following actions:
```
Comment (kyriathar) [inline content/en/platform/corda/5.0/developing-applications/packaging/code-signing/signing-cli.md:29]:
```suggestion
2. Build a CPI (version 2). The group policy file can be passed into the CPI by either passing it to the cli package command parameters as a file or by piping it to the cli package command as shown below:
```

Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/developing-applications/packaging/code-signing/signing-cli.md]: Updated according to your suggestions @kyriathar

**Review by kyriathar — COMMENTED** [review]
This looks good to me but I'd like to wait for @anien  feedback.

**Review by anien — APPROVED** [review]
Looks good to me. Thanks @kyriathar for suggestion.

**Review by Niamh25 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @AnnaK-R3

**Pull request merged**